code_schemes/s01e02.json

```diff
@@ -0,0 +1,71 @@
+{
+  "SchemeID": "Scheme-52ec42a4",
```

Comment:
This is a new scheme, so shouldn't have the same id as was used in ADSS.

(The demographics, which are the same however, should use the same id)

Reply:
Yes, its good to standardize . Updated!